snapcraft/internal/remote_build/_launchpad.py

```diff
@@ -421,11 +421,6 @@ def _download_snap(self, build: Dict[str, Any]) -> None:
         for url in urls:
             snap_name = _get_url_basename(url)
 
-            # Sanity check file name.
-            if not snap_name.endswith("_{}.snap".format(arch)):
```

Comment:
good... if anything you can keep a startswith(snap_name) (snap_name being the snap_name and not the file_name)

Reply:
The last time I looked at this I didn't know if there were any other possible artifacts that we should be ignoring.  Though I haven't come across anything yet...  We could:
(1) filter for `*.snap`?
(2) just download it all?  if something else did get shipped, it's probably a good idea to download it, because it's about to get deleted...

Reply:
went for #2 and just updated the log handling